pkg/api/controller.go

```diff
@@ -237,10 +237,6 @@ func (c *Controller) DeleteObjects(w http.ResponseWriter, r *http.Request, body
 }
 
 func (c *Controller) Login(w http.ResponseWriter, r *http.Request, body LoginJSONRequestBody) {
-	err := clearSession(w, r, c.sessionStore, InternalAuthSessionName)
-	if err != nil {
-		c.Logger.WithError(err).Error("failed to logout previous session")
-	}
```

Comment:
This used to clear the oidc cookie, here... https://github.com/treeverse/lakeFS/pull/5233/files#diff-672be0187484cee0bc614f3f370d1fe732434bcc65e25c9ffd89a1abf6cd43c1L264

I think we don't need to do that here, because the client now clears its own cookies.